Neos.Flow/Classes/Command/ResourceCommandController.php

```diff
@@ -229,26 +229,28 @@ public function cleanCommand()
         $this->output->progressFinish();
         $this->outputLine();
 
-        if (count($brokenResources) > 0) {
-            if ($mediaPackagePresent) {
-                /* @var AssetRepository $assetRepository */
-                $assetRepository = $this->objectManager->get(AssetRepository::class);
-                /* @var ThumbnailRepository $thumbnailRepository */
-                $thumbnailRepository = $this->objectManager->get(ThumbnailRepository::class);
-            }
+        if (!class_exists(AssetRepository::class) || !class_exists(ThumbnailRepository::class)) {
+            // @todo flow has no dependency on Neos.Media. This code should be extracted.
+            $this->outputLine('Neos.Media is not installed. Skipping cleanup of assest and thumbnails.');
+            $this->quit();
+        }
```

Comment:
Not correct. This skips the cleanup completely, but that's not what code did so far.

Reply:
upsi good catch yes adjusted as requested